projects/modular_llm/train_experts.py

```diff
@@ -200,7 +201,8 @@ def upload_library(expert_library, module):
                 if isinstance(module, MoEModule):
                     with expert_library.batched_commit():
                         for expert_name in module.experts_names:
-                            expert = module.get_expert_instance(expert_name)
+                            expert: Expert = module.get_expert_instance(expert_name)
+                            expert.expert_info.training_config = args
```

Comment:
remove this, not a good idea to store complex object in training_config, it will be transformed to a Dict in the next PR :)